string name
navigation_yeon/RobotPose pose
